code/modules/mob/mob_helpers.dm

```diff
@@ -461,10 +454,10 @@
 
 	var/list/timestamped_message = list("\[[time_stamp()]\] [key_name(src)] [loc_name(src)] (Event #[LAZYLEN(logging[smessage_type])])" = colored_message)
 
-	logging[smessage_type] += timestamped_message
+	LAZYADD(logging[smessage_type], timestamped_message)
```

Comment:
Are you sure this works?

`logging[smessage_type]` is now going to be null, and `null + x == x`, meaning you're just going to be concatenating the strings together. This doesn't make any sense to me.